src/Resources/config/services.xml

```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+
+<container xmlns="http://symfony.com/schema/dic/services" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+           xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd">
+
+    <services>
+        <service id="bitbag.mercanet_bnp_paribas.form.type.gateway_configuration"
+                 class="BitBag\MercanetBnpParibasPlugin\Form\Type\MercanetBnpParibasGatewayConfigurationType"
+                 public="true">
```

Comment:
Why the services are public? If there is no reason, we shouldn't do this.

Reply:
It's required to pass behat tests.